abap-code-review/README.md

```diff
@@ -0,0 +1,25 @@
+# ABAP Code Review Guide
+ABAP Code Review Guide is a collection of ideas, tools, and approaches for implementing ABAP code reviews in various landscapes.
+
+The document is provided as open source under the [CC license](../LICENSE), suggestions and bugfixes are welcome.
+
+Latest release from main can be found out [abap-code-review-guide.pdf](https://github.com/SAP/styleguides/releases/download/latest/abap-code-review-guide.pdf).
+
+## Building Locally
+The PDF file can be built locally, if [Node.js](https://nodejs.org/en/) is installed, by running
+
+`npm install && npm test`
+
+## Editing Locally
+The document is written in [AsciiDoc](https://asciidoc.org), and can be edited in any text editor.
```

Comment:
Could you add a few sentences why AsciiDoc has been chosen instead of Markdown.

Reply:
updated, https://github.com/xtough/styleguides/commit/383e88c89ed976a8ec4dc20dd24e4176a722cc6c

Reply:
also see https://github.com/SAP/styleguides/issues/5